app/models/graph/types/species.rb

```diff
@@ -0,0 +1,57 @@
+module Graph
+  module Types
+    Species = GraphQL::ObjectType.define do
+      name "Species"
+      description "A type of person or character within the Star Wars Universe."
+
+      field :id, types.ID, "The ID of this species."
+
+      field :name, types.String, "The name of this species."
+      field :classification, types.String, "The classification of this species, such as \"mammal\" or \"reptile\"."
+      field :designation, types.String, "The designation of this species, such as \"sentient\"."
+
+      field :averageHeight, types.Float do
+        description "The average height of this species in centimeters."
+        property :average_height
+      end
+
+      field :averageLifespan, types.Int do
+        description "The average lifespan of this species in years, null if unknown."
+        property :average_lifespan
+      end
+
+      field :eyeColors, types[types.String] do
+        description "Common eye colors for this species, null if this species does not typically have eyes."
+        resolve ->(species, _, _) do
+          return unless colors = species.eye_colors
+          return if species.eye_colors.include?("none")
```

Comment:
If we don't care about `"none"`, I'll fix the import script to simply omit this value so we can :fire: this line.